Sources/Sentry/SentryOptions.m

```diff
@@ -30,6 +30,7 @@ - (instancetype)init
         self.attachStacktrace = YES;
         self.maxAttachmentSize = 20 * 1024 * 1024;
         self.sendDefaultPii = NO;
+        self.sampleRate = 0;
```

Comment:
`h`: Shouldn't this be 1?

Reply:
For now, that we don't have automatic tracing, probably yes. Other languages use nullable bool to check whether it was changed. Objc does not have nullable bool. A different approach should be discussed here.

Reply:
We should not opt-in by default with 100% sample rate particularly in the first release but probably ever. We can add to the wizard how to set this and what this value means.

By default this is 0 (I'd say it should be null even).